tests/test_decryption_mediator.py

```diff
@@ -106,7 +111,9 @@ def setUp(self):
         builder.set_commitment_hash(self.joint_public_key.commitment_hash)
         self.internal_manifest, self.context = get_optional(builder.build())
 
-        self.encryption_device = EncryptionDevice("location")
+        self.encryption_device = EncryptionDevice(
+            generate_device_uuid(), "Session", 12345, "Location"
```

Comment:
Maybe this should be in the mock section since it is repeated so much?

ElectionFactory.get_encryption_device() ?

Reply:
i was thinking the same thing actually